File: project/angular_project/src/app/models/department.ts
export class Department {
	
  department_id: number;
  department_name: string;
  budget: number;
  description: string;
  contact_info: string;
  head_of_department: string;

  constructor() {
    this.department_id = 0;
    this.department_name = '';
    this.budget = 0;
    this.description = '';
    this.contact_info = '';
    this.head_of_department = '';
  }
}

File: project/src/main/java/com/example/project/entities/Department.java
package com.example.project.entities;


import java.io.Serializable;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;

@Data
@Document(collection = "Department")
public class Department implements Serializable{
	@Id
	private int department_id;
	private String department_name;
	private int budget;
	private String description;
	private String contact_info;
	private String head_of_department;

}
